src/Common/Commands.Common/MetricHelper.cs

```diff
@@ -266,6 +270,12 @@ private void PopulatePropertiesFromQos(AzurePSQoSEvent qos, IDictionary<string,
             eventProperties.Add("HashMacAddress", HashMacAddress);
             eventProperties.Add("PowerShellVersion", PSVersion);
             eventProperties.Add("Version", AzurePowerShell.AssemblyVersion);
+            eventProperties.Add("CommandParameterSetName", qos.ParameterSetName);
+            if (!string.IsNullOrWhiteSpace(qos.InvocationName))
```

Comment:
@praries880 why do we need this if-statement? I didn't think it was possible for `InvocationName` to be null or empty

Reply:
well, now I know!!!